backend/scripts/kmeans_length.py

```diff
@@ -0,0 +1,101 @@
+from sklearn.cluster import KMeans
+from sklearn.utils.testing import ignore_warnings
```

Comment:
I ran into the issue [ModuleNotFoundError: No Module named 'sklearn.utils._testing'](https://stackoverflow.com/questions/62336142/modulenotfounderror-no-module-named-sklearn-utils-testing) due to my version of `sklearn`

As long as the final hosting environment is not throwing this error, it's all good.